libres/lib/enkf/enkf_fs.cpp

```diff
@@ -231,15 +228,19 @@ enkf_fs_type *enkf_fs_alloc_empty(const char *mount_point) {
         util_abort("%s: fatal internal error: mount_point == NULL \n",
                    __func__);
     {
-        char **path_tmp;
-        int path_len;
+        std::vector<std::string> path_tmp;
+
+        ert::split(fs->mount_point, UTIL_PATH_SEP_STRING[0],
+                   [&path_tmp](auto t) { path_tmp.emplace_back(t); });
+
+        int path_len = path_tmp.size();
```

Comment:
The following is perhaps a bit verbose, but should demonstrate the possibility of using the function rather than emplacing on a vector:
```cpp
stringstream root_path_stream;
bool first = true;
ert::split(fs->mount_point, UTIL_PATH_SEP_STRING[0],
                   [&first](auto t) {
    if(!first)
        root_path_stream << UTIL_PATH_SEP_STRING;
    first = false;
    root_path_stream << t;
    fs->case_name = t;
});
fs->root_path = root_path_stream.str()
```

Reply:
I'm probably just used to it by now, but I find the version with `emplace_back` easier to read.
It would of course be better if the function just returned a list of strings :)